USAGE.md

```
@@ -0,0 +1,200 @@
+```
+
+### 3. 実行
+
+```bash
+npx tsx scripts/generate.ts
+```
+
+## AutoCodeOption
+
+| プロパティ | 型 | 必須 | 説明 |
+|-----------|-----|------|------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)